### PR TITLE
Docs: fix some ARIA Authoring Practices Guides broken links

### PR DESCRIPTION
**Pull request by julien-deramond**
This PR fixes some ARIA Authoring Practices Guides broken links found in the documentation and tries to improve consistency in the text of those links.

I don't have access to the previous content so I'm not 100% sure that the modification is well `https://www.w3.org/TR/wai-aria-practices/#something` → `https://www.w3.org/WAI/ARIA/apg/patterns/something/`.

Feel free to comment or directly modify the links.

**Note:** if merged would need to be backported to v4

### Live previews

* [Breadcrumb - Accessibility](https://deploy-preview-36488--twbs-bootstrap.netlify.app/docs/5.2/components/breadcrumb/#accessibility)
* [Collapse - Accessibility](https://deploy-preview-36488--twbs-bootstrap.netlify.app/docs/5.2/components/collapse/#accessibility)
* [Modal - Usage - Via data attributes - Dismiss](https://deploy-preview-36488--twbs-bootstrap.netlify.app/docs/5.2/components/modal/#dismiss)
* [Nav & Tabs - Regarding accessibility](https://deploy-preview-36488--twbs-bootstrap.netlify.app/docs/5.2/components/navs-tabs/#regarding-accessibility)
* [Nav & Tabs - JavaScript behavior - Accessibility](https://deploy-preview-36488--twbs-bootstrap.netlify.app/docs/5.2/components/navs-tabs/#accessibility)
* [Offcanvas - Usage - Via data attributes - Dismiss](https://deploy-preview-36488--twbs-bootstrap.netlify.app/docs/5.2/components/offcanvas/#dismiss)

### CODE REVIEW

**Comment by patrickhlauke** [comment]
Ah, good point ... I spotted an old link earlier while looking at https://github.com/twbs/bootstrap/pull/36487 and then promptly forgot...

File: site/content/docs/5.2/components/breadcrumb.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ $breadcrumb-divider: none;
 
 Since breadcrumbs provide a navigation, it's a good idea to add a meaningful label such as `aria-label="breadcrumb"` to describe the type of navigation provided in the `<nav>` element, as well as applying an `aria-current="page"` to the last item of the set to indicate that it represents the current page.
 
-For more information, see the [WAI-ARIA Authoring Practices for the breadcrumb pattern](https://www.w3.org/TR/wai-aria-practices/#breadcrumb).
+For more information, see the [WAI-ARIA Authoring Practices for the breadcrumb pattern](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/).
```
Comment (patrickhlauke) [inline site/content/docs/5.2/components/breadcrumb.md:97]: suggest changing the wording consistently for all these to something like `ARIA Authoring Practices Guide breadcrumb pattern` or similar

File: site/content/docs/5.2/components/navs-tabs.md [hunk 2]
```diff
@@ -540,7 +540,7 @@ And with vertical pills. Ideally, for vertical tabs, you should also add `aria-o
 
 Dynamic tabbed interfaces, as described in the [<abbr title="Web Accessibility Initiative">WAI</abbr> <abbr title="Accessible Rich Internet Applications">ARIA</abbr> Authoring Practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel), require `role="tablist"`, `role="tab"`, `role="tabpanel"`, and additional `aria-` attributes in order to convey their structure, functionality, and current state to users of assistive technologies (such as screen readers). As a best practice, we recommend using `<button>` elements for the tabs, as these are controls that trigger a dynamic change, rather than links that navigate to a new page or location.
```
Comment (patrickhlauke) [inline site/content/docs/5.2/components/navs-tabs.md:541]: there's an old aria practices link here too

File: site/content/docs/5.2/components/navs-tabs.md [hunk 3]
```diff
@@ -540,7 +540,7 @@ And with vertical pills. Ideally, for vertical tabs, you should also add `aria-o
...
 
-In line with the ARIA Authoring Practices pattern, only the currently active tab receives keyboard focus. When the JavaScript plugin is initialized, it will set `tabindex="-1"` on all inactive tab controls. Once the currently active tab has focus, the cursor keys activate the previous/next tab, with the plugin changing the [roving `tabindex`](https://www.w3.org/TR/wai-aria-practices-1.2/#kbd_roving_tabindex) accordingly. However, note that the JavaScript plugin does not distinguish between horizontal and vertical tab lists when it comes to cursor key interactions: regardless of the tab list's orientation, both the up *and* left cursor go to the previous tab, and down *and* right cursor go to the next tab.
```
[the remark this answers is not in the document]
Comment (patrickhlauke) [reply site/content/docs/5.2/components/navs-tabs.md:543]: they've hidden stuff about roving tabindex here https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/

**Review by patrickhlauke — CHANGES_REQUESTED** [review]

**Review by patrickhlauke — CHANGES_REQUESTED** [review]
Sorry last nitpick

**Review by patrickhlauke — APPROVED** [review]

**Comment by romodog** [comment]
lissten im gonna decode with that thing u linked on opensource and then i learn basics ive been taken away information about this shit for purpuse of u fucking with me my hole life so take it easy im fast learner
________________________________
Från: Julien Déramond ***@***.***>
Skickat: den 2 juni 2022 22:15
Till: twbs/bootstrap ***@***.***>
Kopia: Subscribed ***@***.***>
Ämne: Re: [twbs/bootstrap] Docs: fix some ARIA Authoring Practices Guides broken links (PR #36488)


@julien-deramond commented on this pull request.

________________________________

In site/content/docs/5.2/components/navs-tabs.md<https://github.com/twbs/bootstrap/pull/36488#discussion_r888374620>:

> @@ -540,7 +540,7 @@ And with vertical pills. Ideally, for vertical tabs, you should also add `aria-o

 Dynamic tabbed interfaces, as described in the [<abbr title="Web Accessibility Initiative">WAI</abbr> <abbr title="Accessible Rich Internet Applications">ARIA</abbr> Authoring Practices 1.2](https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel), require `role="tablist"`, `role="tab"`, `role="tabpanel"`, and additional `aria-` attributes in order to convey their structure, functionality, and current state to users of assistive technologies (such as screen readers). As a best practice, we recommend using `<button>` elements for the tabs, as these are controls that trigger a dynamic change, rather than links that navigate to a new page or location.

-In line with the ARIA Authoring Practices pattern, only the currently active tab receives keyboard focus. When the JavaScript plugin is initialized, it will set `tabindex="-1"` on all inactive tab controls. Once the currently active tab has focus, the cursor keys activate the previous/next tab, with the plugin changing the [roving `tabindex`](https://www.w3.org/TR/wai-aria-practices-1.2/#kbd_roving_tabindex) accordingly. However, note that the JavaScript plugin does not distinguish between horizontal and vertical tab lists when it comes to cursor key interactions: regardless of the tab list's orientation, both the up *and* left cursor go to the previous tab, and down *and* right cursor go to the next tab.


I haven't found a new link replacing https://www.w3.org/TR/wai-aria-practices-1.2/#kbd_roving_tabindex

**Pull request merged**